context.go

```diff
@@ -216,7 +251,7 @@ func (context *Context) Close() {
 	defer context.mutex.Unlock()
 
 	wafLib.WafContextDestroy(context.cContext)
-	unsafe.KeepAlive(context.cgoRefs) // Keep the Go pointer references until the end of the context
+	unsafe.KeepAlive(context.cgoRefs) // Keep the Go pointer references until the max of the context
```

Comment:
"The max of the context" does not mean much here...

Maybe:
```suggestion
	unsafe.KeepAlive(context.cgoRefs) // Keep the Go pointer references alive as long as the context is usable
```